pubspec.yaml

```diff
@@ -53,6 +54,7 @@ dependencies:
   timezone: ^0.9.0
   tuple: ^2.0.0
   universal_html: ^2.0.8
+  universal_io: ^2.2.2
```

Comment:
What is this used for ?

Reply:
This is used to send HTTP requests from mobile and web. Maybe I could try to use `universal_html` instead.

Reply:
Yes, since it is already used, I think there is no need to duplicate packages